README.md

```diff
@@ -42,21 +39,36 @@ You need to classify also the psick class from the psick module (which by defaul
 specific Hiera data it does nothing) in order to leverage on the general variables
 set in the main psick class and used in the psick modules.
 
+In short in you manifests/site.pp or wherever you classify your nodes you need to:
+
+    include psick
+
+and then include/classify the psick profiles you want, for example:
+
+    include psick_profile::gitlab
+    include psick_profile::grafana
+
+
 ### What psick_profile affects
 
 Every psick profile manages the relevant application.
 
-In some cases you have the option if to use an external component module to install it
+In some cases you have the option to decide if to use an external component module to install it
 or use Tiny Puppet.
 
-Refere to each profile documentation for more info on the managed resources.
+Refer to each profile documentation for more info on the managed resources.
+
+The classes when an application is installed using Tiny Puppet are always called tp and are placed in manifests called tp.pp
+
+For example the class `psick_profile::jenkins::tp`, defined under `manifests/jenkins/tp.pp` manages with installation of Jenkins via Tiny Puppet and is, by defauly, included from the main `psick_profile::jenkins` class.
+
 
 ### Setup Requirements
 
 Psick_profile module requires:
 
 -   example42-psick module
--   example42-tp module (which need the example42-tinydata module)
+-   example42-tp module (which needs the example42-tinydata module)
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/example42/puppet-psick_profile/pullRequest?prid=9480555)
```suggestion
- example42-tp module (which needs the example42-tinydata module)
```